build/make.go

```diff
@@ -450,3 +458,51 @@ func getPackageArchSuffix() string {
 	}
 	return "x86_64"
 }
+
+func bumpVersion() {
```

Comment:
instead of manipulating the `version.go` using ast parsing, we can externalize the version either as a text file (eg. json) or use env or even use ld flags.